### ISSUE TITLE
Cannot ping 8.8.8.8 with example thread_border_router (TZ-1479)

### ISSUE THREAD

**Issue by FMistrorigo** [espressif/esp-thread-br#120]
### Checklist

- [x] Checked the issue tracker for similar issues to ensure this is not a duplicate.
- [x] Provided a clear description of your suggestion.
- [x] Included any relevant context or examples.

### Issue or Suggestion Description

Hi, 
I'm working with this example of a Thread Border Router using an ESP32-S3 and an ESP32-H2 devkit. 
Everything seems fine, i have the correct esp_idf version and the v1.1 version of esp-thread-br, but while I've managed to accomplish many tasks, I'm struggling to ping the server 8.8.8.8 using CLI commands. 

I followed these steps:
- cloned the directory of esp-idf (commit c8fc5f643b7a7b0d3b182d3df610844e3dc9bd74 - tag v5.3.1)
- cloned the directory of esp-thread-br (commit 50430813bae6e8aacb79c212da570c6e70bfbc9e -  tag: v1.1)
- built the project ot-rcp inside the esp-idf folder (esp-idf/examples/openthread/ot_rcp)
- done the set-target esp32s3 in the basic_thread_border_router folder and modified the menuconfig adding the autostart and the wifi SSID and PW.
- built the project

At this point I was able to use the OpenThread CLI, 
- I checked the wifi state and confirmed it was online.
- I checked that the device was behaving as a leader in my thread network.
- I "used netdata show", "ipaddr" and "br nat64prefix" to check some information about the BR and the Thread Network.
- I used "dataset active -x" to obtain the dataset to add a child device running the cli example in the same network.

Everything is working till this point, but I wanted to check if I was able to ping the address 8.8.8.8 but it didn't work.

>ping 8.8.8.8
Pinging synthesized IPv6 address: fd75:76d8:563d:2:0:0:808:808
1 packets transmitted, 0 packets received. Packet loss = 100.0%.
Done

The Address is being synthetized correctly with the NAT64 prefix but I cannot recieve any packet, the wifi connection has access to Internet and I was able to ping 8.8.8.8 from a Child device using an Apple Border Router.

Could you please help me troubleshoot this issue?

Here's some Logs of everything I did once the BR started:
[Thread_BR_LOGS.txt](https://github.com/user-attachments/files/18440399/Thread_BR_LOGS.txt)
